.gitignore

```diff
@@ -47,3 +47,5 @@ schema-latest.graphql
 
 ci-share/
 rest-media-temp/
+
+docs/build/
```

Comment:
Let's remove the files from git as well.
![2023-06-20-111652](https://github.com/the-deep/server/assets/7059255/ed7988d8-16b0-4f93-870b-c781bdda99bc)